mod/missions/views/default/page/elements/mission-manager-info.php

```diff
@@ -14,7 +14,7 @@
 $container_class = $vars['container_class'];
 $grid_number = $vars['grid_number'];
 $test = $mission->account;
-$manager_account = get_user($mission->account); //Nick changed to owner_guid then back to account
+$manager_account = get_user($mission->owner_guid); //Nick changed to owner_guid then back to account
```

Comment:
I remember the issue this caused by using guid (I just noticed my comment). Sometimes users will create a mission on behalf of their manager. So when they create the opportunity they want the contact person to be the manager and this is found by the email input I believe. So it will display the manager as the opportunity owner and not the user who filled out the form.

When we use owner_guid it will get the guid of the creator and they might not be the actual opportunity owner,

Perhaps a test will have to be done when creating the opportunity so that the account metadata contains the guid of the proper opportunity owner based on the email the user put in, and if the email is the same, default to the creator guid.